tf_quant_finance/black_scholes/vanilla_prices.py

```diff
@@ -64,21 +68,41 @@ def option_price(*,
     forwards: A real `Tensor` of any shape that broadcasts to the shape of
       `volatilities`. The forwards to maturity. Either this argument or the
       `spots` must be supplied but both must not be supplied.
+    discount_rates: An optional real `Tensor` of same dtype as the
+      `volatilities`. If not `None`, discount factors are calculated as e^(-rT),
```

Comment:
Please add the shape specification. Like this

An optional `Tensor` of same dtype as the `volatilities` and of the shape that broadcasts with `volatilities`

The same for dividends and cost of carries

Reply:
done!